src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -149,15 +142,13 @@ public Errors validatePartialUpdateDetails(OverseasEntitySubmissionDto overseasE
         var entityDto = overseasEntitySubmissionDto.getEntity();
 
         if (Objects.nonNull(entityDto)) {
-            var entityEmail = entityDto.getEmail();
-            // Temporary as initial public data Entity fetch has no Email Address. UAR-711
-            if (StringUtils.isNotBlank(entityEmail)) {
-                entityDtoValidator.validate(entityDto, errors, loggingContext);
-            }
+            entityDtoValidator.validate(entityDto, errors, loggingContext);
         }
-        // Temporarily disabling BO/MO validation till it is implemented in Update Journey UAR-711
+
         errors = validatePartialCommonDetails(overseasEntitySubmissionDto, errors, loggingContext);
 
+        ownersAndOfficersDataBlockValidator.validateOwnersAndOfficers(overseasEntitySubmissionDto, errors, loggingContext);
```

Comment:
you could move calls to validateOwnersAndOfficers into validatePartialCommonDetails as done for both update and registration now.